photon-ml/src/main/scala/com/linkedin/photon/ml/util/PhotonLogger.scala

```diff
@@ -63,6 +73,23 @@ protected[ml] class PhotonLogger(logPath: Path, sc: SparkContext) extends Marker
    */
   def close() {
     writer.close()
+    val fs = FileSystem.get(new URI(logPath.toString()), new Configuration())
+
+    val parent = logPath.getParent
+    // Create the parent directory if it doesn't exist already
+    if (!fs.exists(parent)) {
+      fs.mkdirs(parent)
+    }
+
+    // Overwrite any existing file
+    // TODO: make this configurable?
+    if (fs.exists(logPath)) {
+      fs.delete(logPath, false)
+    }
+
+    fs.copyFromLocalFile(tmpLocalPath, logPath)
+
+    new File(tmpLocalPath.toString()).delete()
```

Comment:
Hi @joshvfleming , how are things going?

This is a very good catch, I'll address it : )